dev-libs/openssl/openssl-1.1.1k.ebuild

```diff
@@ -47,12 +47,9 @@ PDEPEND="app-misc/ca-certificates"
 
 PATCHES=(
 	"${FILESDIR}"/${PN}-1.1.0j-parallel_install_fix.patch #671602
-	"${FILESDIR}"/${PN}-1.1.1k-release-changes.patch
```

Comment:
The first commit should delete the whole folder and create it with new upstream contents, this should also delete these patch files which are not needed anymore.

Reply:
@pthos: Do you mean there should be another commit, which reverts the patch commit, then sync with Gentoo, and then apply the Flatcar patches?

Reply:
No, it can be in one commit but before syncing, remove the existing files (first step in https://github.com/kinvolk/coreos-overlay#updating)

Reply:
done